src/main/java/seedu/address/logic/commands/TaskCommand.java

```diff
@@ -0,0 +1,103 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ID;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.Collections;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.StudentId;
+import seedu.address.model.person.StudentIdContainsKeywordsPredicate;
+import seedu.address.model.person.Task;
+import seedu.address.model.person.TaskList;
+
+/**
+ * Displays the list of task that a particular student currently has.
+ */
+public class TaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "task";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all the tasks that are allocated to a particular student. "
+            + "Parameters: "
+            + PREFIX_ID + "STUDENT_ID "
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_ID + "A0123456Z";
+
+    public static final String MESSAGE_SUCCESS = "Here are the tasks that %1$s has:\n\n" + "%2$s";
+    public static final String MESSAGE_EMPTY_TASKLIST = "No task assigned to this student.\n"
+            + "Perhaps you might want to assign a task using the 'assign' command?";
+    public static final String MESSAGE_PERSON_NOT_FOUND = "There is no student with the specified Student ID!";
+
+    private final StudentId studentId;
+
+    /**
+     * Constructs a new TaskCommand.
+     * @param studentId the student ID of the student to be assigned. //TODO
```

Comment:
```suggestion
     * @param studentId the student ID of the student to be assigned.
```

Not sure if you intended to remove the TODO.

Reply:
Oh right thanks for pointing this out!